reference/commands/create.rst

```diff
@@ -59,6 +59,34 @@ The same happens for lockfiles created with ``--lockfile-out`` argument. The loc
   dependencies, you can control their build using the ``--build-test`` argument.
 
 
+Methods execution order
+-----------------------
+
+The ``conan create`` executes :ref:`methods <reference_conanfile_methods>` of a *conanfile.py* in the following order:
+
+#. ``export()``
+#. ``export_sources()``
+#. ``config_options()``
+#. ``configure()``
+#. ``layout()``
+#. ``requirements()``
+#. ``package_id()``
+#. ``validate()``
+#. ``validate_build()``
+#. ``build_requirements()``
```

Comment:
This is not true. The call that calls ``requirements()``: https://github.com/conan-io/conan/blob/06fdb130171640c6191b1cddfb44119d67b326b6/conans/client/conanfile/configure.py#L45